src/main/java/io/cdap/plugin/dlp/SensitiveRecordFilter.java

```diff
@@ -305,6 +361,26 @@ public void validate(FailureCollector collector, Schema inputSchema) {
           }
         }
       }
+
+      if (dlpOverrideEnabled) {
+        if (dlpHost == null || dlpHost.isEmpty()) {
+          collector.addFailure("Custom DLP endpoint is enabled, " +
+                  "but DLP host has not been specified.",
+              "Specify the DLP host.")
+              .withConfigProperty(DLP_HOST);
+        }
+        if (dlpPort == null || dlpPort.isEmpty()) {
```

Comment:
Do you want to do any more validation? maybe checking for valid port ranges?

Reply:
Thanks, good point.